### PR TITLE
Feature/token session

### PR DESCRIPTION
**Pull request by MimiXav**

### CODE REVIEW

**Review by stark-dev — APPROVED** [review]

File: CMakeLists.txt [hunk 1]
```diff
@@ -23,11 +23,14 @@ etn_target(shared ${PROJECT_NAME}
         *.h
     USES_PRIVATE
         cxxtools
-        protobuf        
+        protobuf
 )
 
+# Remove protobuf warning
+target_compile_options(${PROJECT_NAME} PUBLIC "-Wno-useless-cast")
```
Comment (perrettecl) [inline CMakeLists.txt:30]: Could be add in the FLAGS  section of etn_target. See https://github.com/42ity/fty-cmake/blob/master/README.md

File: tests/fty_srr_dto_tests.cpp [hunk 2]
```diff
@@ -625,7 +628,7 @@ TEST_CASE("All the stuff of before")
         try
         {
             std::string restoreQueryJson =R"TEST({
-                    "passphrase":"a","version":"1.0","checksum":"qDYTMxDem+",
+                    "passphrase":"a","version":"1.0","checksum":"qDYTMxDem+", "sessionToken":"",
```
Comment (perrettecl) [inline tests/fty_srr_dto_tests.cpp:631]: Session token should not be in the rest payload

**Review by perrettecl — APPROVED** [review]
Beside the fact that we add session token to the rest payload with is quite ugly (here is no much solution without impacting the Java world as they are using the json from the ui...) it's ok for me.

File: CMakeLists.txt (hunk 1 above)
Comment (MimiXav) [reply CMakeLists.txt:30]: done

**Pull request merged**